vaadin-renderer-flow-parent/vaadin-renderer-flow/src/main/resources/META-INF/resources/frontend/lit-renderer.ts

```diff
@@ -37,28 +37,24 @@ _window.Vaadin.setLitRenderer = (
 ) => {
   // Dynamically created function that renders the templateExpression
   // inside the given root element using Lit
```

Comment:
This comment is no longer attached to `const renderFunction`, but `const callablesCreator`. Consider relocating or removing it.

Reply:
Removed comment